dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/model/ContributionAckFromDrc.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.laa.crime.dces.integration.model;
+
+import java.util.Map;
+
+public record ContributionAckFromDrc(Data data, Map<String, String> meta) {
+    public record Data(int id, Integer maatId, String errorText) {
+    }
+
+    public static ContributionAckFromDrc of(int concorContributionId, String errorText) {
```

Comment:
Academic question: why the "of" method, rather than a constructor with the same parameters?